pydy/tests/test_models.py

```diff
@@ -19,8 +16,8 @@ def test_multi_mass_spring_damper():
 
     sys = multi_mass_spring_damper()
 
-    assert Counter(sys.constants_symbols) == Counter([k0, c0, m0])
-    assert sys.specifieds_symbols == list()
+    assert sys.constants_symbols == {k0, c0, m0}
```

Comment:
What's the procedure for that? I looked through the code and comments and didn't see anything about the list having a specific order. It would have a fixed order once constructed, but it can vary every time you create the the object, as was seen with occasional test failures.